cmd/cm.go

```diff
@@ -34,5 +35,6 @@ var cmCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(cmCmd)
 	cmCmd.PersistentFlags().StringVar(&kubeNamespace, "kube_namespace", "default", "The namespace to get the cm from")
+	cmCmd.PersistentFlags().StringVar(&dataKey, "dataKey", "dataKey", "The key to get the cm from")
```

Comment:
default value should probably be values.yaml to ensure backward compatibility. I see you have that for the secret one.

Reply:
my bad - typoed the cm one 😆 setting it to `values.yaml` like it is on secrets